QStore4S/pom.xml

```diff
@@ -220,8 +220,8 @@
 			<!-- Dev Env. Properties -->
 			<properties>
 				<!-- Neo4j Properties -->
-				<neo4j.user>${neo4j.username}</neo4j.user>
-				<neo4j.pw>${neo4j.password}</neo4j.pw>
+				<neo4j.user>neo4j</neo4j.user>
+				<neo4j.pw>myneo4j</neo4j.pw>
```

Comment:
Please put the variables back in here, or you will break the build for everybody else.